packages/utils/src/tests/test_math.cairo

```diff
@@ -0,0 +1,75 @@
+use core::integer::{u512, u512_safe_div_rem_by_u256};
+use core::num::traits::OverflowingAdd;
+use crate::math::average;
+
+#[test]
+fn test_average_u8(a: u8, b: u8) {
```

Comment:
I noticed that fuzzing actually takes a significant amount of time when running tests, not only in compilation. Did you noticed the same thing?

Reply:
I did...it's only a few tests though so relative to the whole repo, IMO it's not a huge deal. If we want to improve performance, we can configure it to a smaller amount either in the tests themselves or globally when running the test cmd i.e. `snforge test ... --fuzzer-runs 25`